build_resources.yml

```diff
@@ -0,0 +1,24 @@
+# ui_src, py_dest must be absolute paths.
```

Comment:
Can you update this to use just relative paths and tk-toolchain logic can "autodetect" the runtime path so it can be used as path base?